keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -316,7 +327,7 @@ Additionally, privileged containers may impact other pod security policies (PSPs
    </td>
    <td>no
    </td>
-   <td>No support for pid and ipc since jobs don’t provide isolation at that level.
+   <td>Windows does not have configurable PID/IPC namespaces (unlike Linux). Job objects run in the host 'silo' and will have access to other other host processes. Because on Windows this setting is per-container instead of being Pod-wide and also because future plans include the improvements to schedule pods container both normal and 'privileged'/HostProcess containers in the same Pod we will not enforce setting this pod security flag for HostProcess containers.
```

Comment:
```suggestion
   <td>Windows does not have configurable PID/IPC namespaces (unlike Linux). Job objects run in the host 'silo' and will have access to other host processes. Because on Windows this setting is per-container instead of being Pod-wide and also because future plans include the improvements to schedule pods container both normal and 'privileged'/HostProcess containers in the same Pod we will not enforce setting this pod security flag for HostProcess containers.
```